app/components/BlogCard.tsx

```diff
@@ -0,0 +1,75 @@
+import { FunctionComponent } from "react";
+import dayjs from "dayjs";
+import cx from "classnames";
+import { PostOrPage } from "@tryghost/content-api";
+
+type Props = {
+  blogPost: PostOrPage;
+};
+
+const BlogCard: FunctionComponent<Props> = ({ blogPost }) => {
+  const tag = (blogPost.tags || [])[0];
+  const author = (blogPost.authors || [])[0];
+
+  return (
+    <div className="relative flex flex-col w-full" style={{ maxWidth: 400 }}>
+      <div className="flex-grow">
+        <a href={blogPost.url} target="_blank" className="mb-2">
+          <img
+            src={blogPost.og_image || ""}
+            alt="blog-card-img"
+            className="post-image"
+          />
+        </a>
+        <div className="flex">
+          <p
+            className={cx("post-tag", {
+              "bg-indigo-500": tag.visibility === "public",
+            })}
+          >
+            {tag.visibility === "public" ? tag.name : ""}
+          </p>
+        </div>
+        <div className="mt-6">
+          <h1 className="post-title text-lg font-bold text-black-500">
```

Comment:
**suggestion:** it would be nice to make this a link to the article, just like the image.

Reply:
Link added!